app/services/doc_auth/lexis_nexis/requests/true_id_request.rb

```diff
@@ -0,0 +1,69 @@
+module DocAuth
+  module LexisNexis
+    module Requests
+      class TrueIdRequest < DocAuth::LexisNexis::Request
+        attr_reader :front_image, :back_image, :selfie_image, :liveness_checking_enabled
+
+        def initialize(
+          front_image:,
+          back_image:,
+          selfie_image: nil,
+          liveness_checking_enabled: nil
+        )
+          @front_image = front_image
+          @back_image = back_image
+          @selfie_image = selfie_image
+          @liveness_checking_enabled = liveness_checking_enabled
+        end
+
+        private
+
+        def body
+          document = {
+            Document: {
+              Front: encode(front_image),
+              Back: encode(back_image),
+              DocumentType: 'DriversLicense',
+            },
+          }
+
+          document[:Document][:Selfie] = encode(selfie_image) if liveness_checking_enabled
```

Comment:
Just an idea here, to not have to add this `Selfie` key outside, might make it easier to understand the structure of the hash?

```suggestion
          document = {
            Document: {
              Front: encode(front_image),
              Back: encode(back_image),
              Selfie: encode(selfie_image) if liveness_checking_enabled,
              DocumentType: 'DriversLicense',
            }.select { |_, v| v.present? },
          }
```